README.md

```diff
@@ -10,9 +10,9 @@
 Build | Status
 -- | --
 Test coverage | [![codecov](https://codecov.io/gh/gazebosim/gz-plugin/tree/main/graph/badge.svg)](https://codecov.io/gh/gazebosim/gz-plugin/tree/main)
```

Comment:
Can you update "Test coverage" as well?

Reply:
Updated in [915aa4c](https://github.com/gazebosim/gz-plugin/pull/148/commits/915aa4cd7f5cf3612f19c96dc66f7e46c6e20500).